source/whats-new.txt

```diff
@@ -41,9 +41,31 @@ What's New in 5.0
 
 New features of the 5.0 {+driver-short+} release include:
 
-- Removal of ``Collection.insert()``, ``Collection.update()``, and
-  ``Collection.remove()`` helper methods.
+- Optional installation of ``@aws-sdk/credential-providers`` as a dependency.
 
+  - The driver no longer includes AWS SDK modules by default. If you
+    install the SDK, ``npm`` notifies you if the version of the SDK you
+    installed is incompatible with the driver. When the dependency is
+    installed, the driver uses the AWS SDK itself to manage credentials
+    from the environment. Use the following ``npm`` command to install the SDK:
+
+    .. code-block:: bash
+
+       npm install --save @aws-sdk/credential-providers@3.186.0
+
+.. important:: Breaking Change: Removal of CRUD Helper Methods
+
+   This release removes support for the ``Collection.insert()``, ``Collection.update()``, and
+   ``Collection.remove()`` helper methods. The following list describes
+   methods to migrate to:
+
+   - ``insertOne()`` and ``insertMany()`` in place of
+     ``Collection.insert()``
+   - ``updateOne()`` and ``updateMany()`` in place of
+     ``Collection.update()``
+   - ``deleteOne()`` and ``deleteMany()`` in place of
+     ``Collection.remove()``
```

Comment:
[suggestion] For this list, I'd recommend putting the thing users are migrating from first, since that is what they will be looking for first.

For example,
Migrate from ``Collection.insert()`` to either ``insertOne()`` or ``insertMany()``.